jwst/wavecorr/wavecorr.py

```diff
@@ -261,55 +276,18 @@ def get_source_xpos(slit, slit_wcs, lam, msa_model):
     vparity = slit.meta.wcsinfo.vparity
 
     idl2v23 = trmodels.IdealToV2V3(v3idlyangle, v2ref, v3ref, vparity)
+    log.debug("wcsinfo: {0}, {1}, {2}, {3}".format(v2ref, v3ref, v3idlyangle, vparity))
     # Compute the location in V2,V3 [in arcsec]
     xv, yv = idl2v23(xoffset, yoffset)
+    log.info(f'xoffset, yoffset, {xoffset}, {yoffset}, {lam}')
```

Comment:
You're printing 3 values but only 2 of them are named in the text part of the message.

Reply:
fixed